### PR TITLE
docs: added missing functions: [abc]

### LINKED ISSUE #868: Wrong example in docu for "Custom Functions"
**Describe the bug**
I tried to create a custom function as described in the documentation [Custom Functions](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/guides/custom-functions.md#custom-functions) using an own function-directory. 

**To Reproduce**
I copied the `abc.js` file from [Example](https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/guides/custom-functions.md#example) into a directory `.\my-functions` relative to the ruleset file `.spectral.yml`.

In my rules file I added the `functionsDir: ./my-functions` property as described in the last example (section starts with _If for some reason, you do not want..._).

This does not work, because the property `functions: [abc]` is missing in the example.

**Expected behavior**
Documentation should be correct :-)

### PR DESCRIPTION
**Pull request by philsturgeon**
Fixes #868

**Checklist**

- [ ] ~Tests added / updated~
- [x] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: docs/getting-started/openapi.md [hunk 1]
```diff
@@ -4,4 +4,5 @@ Spectral is a generic linter, but a lot of effort has been put in to making sure
 
 Run Spectral against a document without specifying a ruleset will trigger an auto-detect, where Spectral will look to see if `swagger: 2.0` or `openapi: 3.0.x` is at the top of the file. If it finds either of those it will load `spectral:oas2` or `spectral:oas3`, both of which are documented in our [Reference > OpenAPI Rules](../reference/openapi-rules.md).
 
-_**Note:** If you would like support for other API description formats like RAML, message formats like JSON:API, or anything at all, we recommend you start building up custom, but generic rulesets that can be shared with others. These can then be shared around in a sort of marketplace in the future._
+<!-- theme: info -->
+> If you would like support for other API description formats like [RAML](https://raml.org/), message formats like [JSON:API](https://jsonapi.org/), event-driven API descriptions like [AsyncAPI](https://asyncapi.io/), etc., we recommend you start building custom but generic rulesets which can be shared with others. We've started putting together some over here on [OpenAPI Contrib](https://github.com/openapi-contrib/style-guides/)._
```
Comment (P0lip) [inline docs/getting-started/openapi.md]: :heart:

File: docs/guides/custom-functions.md [hunk 2]
```diff
@@ -16,14 +16,18 @@ export type IFunction<O = any> = (
...
+For instance, if `given` has a JSONPath expression of `$`, an the following partial of OpenAPI 3.0 document:
+
 ```yaml
 openapi: 3.0.0
 info:
   title: foo
 ```
-and the following `given` JSONPath expression `$`, `targetValue` would be a JS object literal containing `openapi` and `info` properties.
+
+The  and `targetValue` would be a JS object literal containing `openapi` and `info` properties.
```
Comment (nulltoken) [inline docs/guides/custom-functions.md]: @philsturgeon Aren't we missing a word (or a couple of words) here between `The` and `and`?

File: docs/getting-started/openapi.md (hunk 1 above)
Comment (nulltoken) [reply docs/getting-started/openapi.md]: I like this very much -> https://github.com/openapi-contrib/style-guides/ <3

File: docs/guides/custom-functions.md [hunk 3]
```diff
@@ -126,6 +130,7 @@ module.exports = (targetVal, opts) => {
 If for some reason, you do not want to place your functions in a directory called `functions`, you can specify a custom directory.
 
 ```yaml
+functions: [abc]
```
Comment (nulltoken) [inline docs/guides/custom-functions.md]: @Snoopy1964 Does this fit your request from #868?

File: docs/getting-started/openapi.md (hunk 1 above)
Comment (philsturgeon) [reply docs/getting-started/openapi.md]: @nulltoken thanks! Can we get a +1? Merging to master needs two thumbs.

File: docs/guides/custom-functions.md (hunk 2 above)
Comment (philsturgeon) [reply docs/guides/custom-functions.md]: Oh damn yeah

Comment (philsturgeon) [reply docs/guides/custom-functions.md]: Fixed, can you re-review?

Comment (nulltoken) [reply docs/guides/custom-functions.md]: Looks like there’s still something missing between ´The´ and ´and’ 🧐

File: docs/guides/custom-functions.md (hunk 3 above)
Comment (Snoopy1964) [reply docs/guides/custom-functions.md]: @nulltoken Yup, the correction you made in the example section of the custom-functions.md in the master branch fits!

Nevertheless I can't see it in the official documentation https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/guides/custom-functions.md#example

Thanks a lot!

Comment (nulltoken) [reply docs/guides/custom-functions.md]: @philsturgeon Would you know how often is the main website updated with the doco content? It seems to still reflect the old content.

image

**Comment by philsturgeon** [comment]
Merging and publishing are different things. We had to disable auto-publish for a little while but we can bring it back now. Thanks for bringing this to my attention.

**Pull request merged**